### PR TITLE
Fix OSX implementation of omrsig_info()

### PR DESCRIPTION
**Pull request by keithc-ca**
The code was returning addresses into a _copy_ of parts of the thread state eliminating the possibility that a signal handler might update that state before allowing a thread to resume.

An example modification is in OpenJ9's handler for division by zero which wants to alter the instruction pointer to a trap handler, but writing to that copy of thread state is ineffective.

### CODE REVIEW

**Comment by DanHeidinga** [comment]
@charliegracie @0xdaryl This looks reasonable to me and addresses allows OpenJ9 to take the correct actions (ie: run special trap handlers) in response to signals on mac, just as it does on linux.

Can one of you review this?  We need it for OpenJ9's 0.11.0 release which is imminent.

File: port/osx/omrsignal_context.c [hunk 1]
```diff
@@ -26,7 +26,7 @@
 #include <unistd.h>
 
 void
-fillInUnixSignalInfo(struct OMRPortLibrary *portLibrary, void *contextInfo, struct OMRUnixSignalInfo *j9Info)
+	fillInUnixSignalInfo(struct OMRPortLibrary *portLibrary, void *contextInfo, struct OMRUnixSignalInfo *j9Info)
```
Comment (charliegracie) [inline port/osx/omrsignal_context.c]: why is this tabbed in?

**Comment by 0xdaryl** [comment]
Seems like a reasonable change to me.

**Comment by charliegracie** [comment]
@genie-omr build all

**Pull request merged**